### PR TITLE
fix: header navBar hovering issue

### PR DESCRIPTION
**Pull request by BaskarMitrah**
removed dropdown visible while hover and added checkmark near to show which is being displayed

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by louisachu — CHANGES_REQUESTED** [review]

**Review by louisachu — COMMENTED** [review]
I've checked it against document services.  If you try it on the menu from, I think the checkmark would work better on the left instead on the right?  Because the styling looks off when the width can be variable of the title.  If you goto the last item on the list, the checkmark will be missing. 
Screen Shot 2023-02-02 at 11 38 58 AM

File: packages/gatsby-theme-aio/src/components/GlobalHeader/index.js [hunk 1]
```diff
@@ -760,9 +744,22 @@ const GlobalHeader = ({
...
+                                    margin-top: var(--spectrum-global-dimension-size-50);
+                                    margin-bottom: var(--spectrum-global-dimension-size-50);
+                                    display: flex;
+                                    flex-direction: row;
+                                    justify-content: space-between;
+                                  `}>
+                                    <div>{menu.title}</div>
+                                    {
+                                    location.pathname.endsWith(`/${menu.path}`) &&
+                                          <CheckMarkIcn  css={css`
```
[the remark this answers is not in the document]
Comment (BaskarMitrah) [reply packages/gatsby-theme-aio/src/components/GlobalHeader/index.js]: @louisachu Changed the Icon name

**Comment by BaskarMitrah** [comment]
@louisachu 


> I've checked it against document services. If you try it on the menu from, I think the checkmark would work better on the left instead on the right? Because the styling looks off when the width can be variable of the title. If you goto the last item on the list, the checkmark will be missing. Screen Shot 2023-02-02 at 11 38 58 AM

You are correct,placed the checkmark on the left of the title

**Review by louisachu — COMMENTED** [review]
I ran the branch under the examples in our aio-theme.  It doesn't show any checkmark.  Can you please check that again?  Also, the hover over with the click seems awkward.

**Comment by BaskarMitrah** [comment]
> I ran the branch under the examples in our aio-theme. It doesn't show any checkmark. Can you please check that again? Also, the hover over with the click seems awkward.

I've fixed the issue. checked the variable with menu.pathname previously it was menu.path. Now I got checkmark on local, 

image

**Comment by louisachu** [comment]
Couple things.
1.) Can you also change where the checkmark icon is for the version? 
2.) the location and menu pathname aren't limited to the same.  We may need to thing of a better way to handle this. Also, the pathname often gets changed when there is nested sidebar menu.  
Screen Shot 2023-02-06 at 1 53 43 PM

**Comment by BaskarMitrah** [comment]
> Couple things. 1.) Can you also change where the checkmark icon is for the version? 2.) the location and menu pathname aren't limited to the same. We may need to thing of a better way to handle this. Also, the pathname often gets changed when there is nested sidebar menu. Screen Shot 2023-02-06 at 1 53 43 PM

I've addressed both points. Please check and let me know

**Comment by louisachu** [comment]
I think the multiple path still doesn't work properly.  If you check in the guides/no-breadcrumbs/, it'll always stay with the guide.  Also the guides isn't showing the text aligned left properly.  Can you please double check? 
Screen Shot 2023-02-08 at 12 18 59 PM

**Comment by BaskarMitrah** [comment]
@louisachu Fixed both issues

File: packages/gatsby-theme-aio/src/components/Menu/index.js [hunk 2]
```diff
@@ -42,13 +42,30 @@ const Item = ({ children, isDivider = false, isHighlighted, isSelected, href = '
...
+      <div css={css`display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    overflow: hidden;
+    ${ isHeightUnset && !isHeightUnset ? `margin-top: var(--spectrum-global-dimension-size-50) !important` : ""}
+}`}>
+        <div css={css`
+          width: 20px !important;
+         ${ isHeightUnset ? "" : "height: 15px !important;" }
+         margin-left: 2px !important;
```
Comment (louisachu) [inline packages/gatsby-theme-aio/src/components/Menu/index.js:54]: If you can add a padding-right: 5px would make the icon looks better to have some of space between the words.

Comment (BaskarMitrah) [reply packages/gatsby-theme-aio/src/components/Menu/index.js]: @louisachu  Added padding-right: 5px now it's looking fine

**Pull request merged**